flatcar/developer-guide: exit SDK before running script (#168)
kinvolk/flatcar-docs: docs/reference/developer-guides/sdk-modifying-flatcar.md

### Booting

> `image_to_vm.sh` will also generate a wrapper script to launch a Flatcar VM with qemu. After exiting the SDK, you can boot the VM with:

Reviewer: ```suggestion
`image_to_vm.sh` will also generate a wrapper script to launch a Flatcar VM with qemu. In a new terminal, without entering the SDK, you can boot the VM with:
```
